backend/functions/lib/adapters/types/http.d.ts

```diff
@@ -8,10 +8,7 @@ export interface ApiResponse {
 }
 
 export interface HttpResponse extends Response {
-  data: {
-    content?: [];
-    status?: number;
-  };
+  data: unknown;
```

Comment:
I believe with the removal of the defunct http-client, the `HttpResponse` interface can be removed.